docs/source/advanced/profiler.rst

```diff
@@ -214,6 +214,50 @@ To visualize the profiled operation, you can either:
     python -c 'import torch; print(torch.autograd.profiler.load_nvprof("trace_name.prof"))'
 
 
+XLA Profiler
+============
+
+:class:`~pytorch_lightning.profiler.xla.XLAProfiler` will help you debug and optimize training
+workload performance for your models using Cloud TPU performance tools.
+
+.. code-block:: python
+
+    # by passing the `XLAProfiler` alias
+    trainer = Trainer(..., profiler="xla")
+
+    # or by passing an instance
+    from pytorch_lightning.profiler import XLAProfiler
+
+    profiler = XLAProfiler(port=9001)
```

Comment:
@kaushikb11 I updated the port from 9012 to 9001 here in 37b1601. Correct? If not, please revert the commit...

Reply:
It's right!